ml-api-adapter/chart-handler-notification/configs/default.json

```diff
@@ -3,6 +3,7 @@
 {
     "PORT": {{ .Values.service.internalPort }},
     "ENDPOINT_SOURCE_URL": "http://{{ $centralServicesHost }}:{{ .Values.config.central_services_port }}/participants/{{"{{"}}fsp{{"}}"}}/endpoints",
```

Comment:
Please revert the missing `{`

Reply:
Or is this being high-lighted incorrectly?

Reply:
I think it's just Github syntax highlighting as it's not technically valid JSON.

The change is on line 6.